### DISCUSSION TITLE
Order data JSON export contains no customer info

### DISCUSSION THREAD

**Discussion by ginkel** [pretix/pretix#4446] [Q&A] [answered]
Hi there,

we are using pretix 2024.8.0. I'd like to create an order data export based on JSON (at `/control/event/<organizer>/<event>/orders/export/?identifier=json`) to post-process the order data for a custom report. Unfortunately, the customer field of all order is `null`. Is there a way to get the customer info with this export?

Thanks,
Thilo

**Comment by pc-coholic** [comment]
Just to make sure, that we're not chasing a ghost: Can you verify if any of your orders are indeed linked to a customer account?

See screenshot attached; the section for a customer should be visible on the order page and a customer account selected...

![Screenshot_20240905_232833.jpg](https://github.com/user-attachments/assets/16ab26a1-8ea7-4c7d-b59e-1ae80384798e)

**Reply by ginkel** [reply]
Ah, no, they aren't. We're operating pretix w/o customer accounts. In that case, how do I get customer name and address from the invoice data section (German: "Rechnungsinformationen") exported?

**Reply by pc-coholic** [reply, marked as answer]
Now that you mention it - why don't we expose the invoice address in the json export :thinking: 

For now, your best bet would probably to either pull the information from the API or to use the `orderlist` export - although that won't be json...

**Reply by ginkel** [reply]
Thanks! Using the API works like a charm! Still, adding the invoice address to the JSON export could be valuable for others.
